src/tools/genpolicy/src/registry_containerd.rs

```diff
@@ -359,74 +347,41 @@ async fn get_verity_hash(
     Ok(verity_hash)
 }
 
-async fn create_verity_hash_file(
```

Comment:
No longer need create_verity_hash_file after cache improvements. Now we call create_decompressed_layer_file directly https://github.com/microsoft/kata-containers/pull/113/files#diff-754926dd5feb413b73f033fd1484312752a69e1b24d71fc57e422316b90c587bL329